openpype/hosts/fusion/plugins/create/create_saver.py

```diff
@@ -137,12 +142,19 @@ def _imprint(self, tool, data):
 
     def _update_tool_with_data(self, tool, data):
         """Update tool node name and output path based on subset data"""
-        if "subset" not in data:
+
+        subset = data.get("subset")
+        if subset is None:
             return
 
-        original_subset = tool.GetData("openpype.subset")
-        subset = data["subset"]
-        if original_subset != subset:
+        original_data = tool.GetData("openpype")
+
+        # As data contains an extra key called active, add it to
+        # original_data to get a correct comparison
+        if data.get("active"):
```

Comment:
Shouldn't this be like this? If `"active"` would be `False` it would not change it. (maybe it's desired, but from the comment it is not)
```suggestion
        if "active" in data:
```

Reply:
This note should still be applied?